test/CSharpPrecompiledTestProjects/WebJobsStartupTests/Function1.cs

```diff
@@ -93,7 +93,7 @@ private static bool ValidateConfig(IConfiguration _config)
         {
             if (_config is ConfigurationRoot root)
             {
-                if (root.Providers.Count() != 7)
+                if (root.Providers.Count() != 8)
```

Comment:
Also need to adjust the return statement lines 103-110 to expect this now provider.

Reply:
done